common.blocks/select/select.bemhtml

```diff
@@ -21,7 +21,7 @@ block('select')(
                         iterateOptions(option.group);
                     } else {
                         firstOption || (firstOption = option);
-                        optionIds.push(option.id = _this.generateId());
+                        optionIds.push(option.id = 'option' + random++);
```

Comment:
If we have `group`, the `i` iterator will starts from zero. `id` attribute will not be unique in this case.